src/routes/healthcheck/healthcheck-get.ts

```diff
@@ -1,8 +1,22 @@
 import { Request, Response } from "express";
 import { getLogger } from "config/logger";
+import { envVars } from "config/env";
+import { deepCheckCryptor } from "./deep-check-cryptor";
 
 const logger = getLogger("healthcheck");
-export const HealthcheckGet = (_: Request, res: Response): void => {
+export const HealthcheckGet = async (_: Request, res: Response): Promise<void> => {
+
+	if (envVars.MICROS_GROUP === "Worker") {
+		try {
+			await deepCheckCryptor();
+		} catch (e) {
+			logger.warn("Cryptor not ready yet", { error: e });
```

Comment:
`Cryptor not ready yet` could be confusing if its a legit Cryptor error, once it is ready.

Reply:
I can't think of any way to distinguish whether it is just not ready or it is ready but has error. I can do a different text, like `Cryptor not ready yet or has error`? You reckon that's okay?

Reply:
Can't we just use something more generic like `Cryptor deepcheck failed`? That plus the error should give enough info without being misleading.

Reply:
thanks team, fixed per @rachellerathbone suggestion.